### ISSUE TITLE
Перенос календаря в другой формат и добавить других лиц к задаче

### ISSUE THREAD

**Issue by MilaMaric** [cyberFund/golosfund#76]
А что скажете если мы поменяем на более доступный формат в excell? Предлагаю расшерить со всеми эту задачу? чем больше глаз тем больше конференций про которые мы будем знать

**Comment by serejandmyself** [comment]
Ты имеешь ввиду перенос чего? Календаря ивентов?

**Comment by MilaMaric** [comment]
Да, из общедоступного календаря, где не очень понятно как все выглядит в одну таблицу one pager
